data-exporter/src/test/java/org/reactome/release/dataexport/NCBIEntryTest.java

```diff
@@ -64,7 +64,7 @@ public void retrievesNCBIEntryFromDummyGraphDB() {
 
 		List<NCBIEntry> ncbiEntries = NCBIEntry.getUniProtToNCBIGeneEntries(dummyGraphDBServer.getSession());
 
-		assertThat(ncbiEntries, contains(expectedNCBIEntry));
+		assertThat(ncbiEntries, hasItem(expectedNCBIEntry));
```

Comment:
I believe that `contains` is the more stringent matcher. What was the reason for switching to `hasItem`?

Reply:
That's true - `contains` is more stringent but it requires the `expectedNCBIEntry` to be the only entry in `ncbiEntries`.  Another entry was added in the mock data to include the case of a UniProt isoform, so it was necessary to change to `hasItem` so the test would succeed with only one of the two entries being checked for.